config/routes.rb

```diff
@@ -75,6 +75,9 @@
 
         get '/review' => 'degrees/review#show', as: :degrees_review
         patch '/review' => 'degrees/review#complete', as: :degrees_complete
+
+        get '/delete/:id' => 'degrees/destroy#confirm_destroy', as: :degrees_destroy
```

Comment:
what about `as: :confirm_degrees_destroy`? is a bit longer but it's more descriptive